.azure-pipelines/baseline_test/baseline.test.mgmt.public.master.yml

```diff
@@ -28,6 +28,13 @@ stages:
   dependsOn:
   - Pre_test
   condition: succeeded('Pre_test')
+  variables:
+  - group: SONiC-Elastictest
+  - name: inventory
+    value: veos_vtb
+  - name: testbed_file
+    value: vtestbed.yaml
+  - group: GIT_SECRETS
```

Comment:
All Test_round_1, Test_round_2 and Test_round_3 have the same variables, can the definition of these variables be extracted to parent index?

Reply:
No, even defined in previous step, we also need to import to different stages, maybe we can create a template file and import template, but I think it's unnecessary, the repeat code is not much

Reply:
I'm not quite sure, can't we define the variables in parent stage? Like this:
https://github.com/sonic-net/sonic-buildimage/blob/eba6ef0aa9a15f72ffdc83045f5d17730a9a7ed3/azure-pipelines.yml#L86

Reply:
This is defined in stage level, just like I defined

Reply:
Oh, my bad, I regard it as job carelessly.